cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax.asciidoc

```diff
@@ -16,9 +16,9 @@ SHOW USERS
 
 | [source, cypher]
 ----
-SHOW USER [name] PRIVILEGES
+SHOW USER[S] [name1[, name2]] PRIVILEGES
 ----
-| List the privileges granted to a user.
+| List the privileges granted to the specified users, or the current user if omitted.
```

Comment:
Do you think that it would be clearer by writing something like "...or the current user, if no user is specified"? To me it's not clear what can be omitted right now.

Reply:
The `[...]` defines it as optional, and here its for the whole `[name1[, name2]]` but I agree that it can be confusing, will update :)